subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/README.md

```diff
@@ -1,24 +1,43 @@
+
+HiveD allows flexible cell definitions and assignments. For example, users can define cells at multiple topology levels (e.g., PCI-e switch) for finer-grained allocations. Users can also create different cell types for different GPU models (e.g., V100) or networking configuration (e.g., InfiniBand domain) in a heterogeneous cluster.
+
+### Gang Scheduling
+
+HiveD optimizes the performance of **[gang scheduling](example/feature/README.md#Gang-Scheduling)**, a typical scheduling requirement for deep learning training jobs, where all containers should be allocated before the training job can begin. Multiple gang-scheduled jobs competing for the same set of resource may lead to starvation, where each job only gets partial resource and has to wait indefinitely.
+
+HiveD schedules all containers within a job in a *transactional* manner, i.e., all these containers' requirements will be granted or denied as a whole, thus avoiding partial resource allocation and starvation.
+
+### Priorities
+
+HiveD supports multiple job **priorities**. Higher-priority jobs can **[preempt](example/feature/README.md#Intra-VC-Preemption)** lower-priority jobs. HiveD also introduces **[opportunistic jobs](example/feature/README.md#Opportunistic-Job)**, i.e., jobs with the lowest priority which can use other VCs' free resource when possible (without breaking the resource guarantees to other VCs).
+
+### Fault-Tolerance
+
+HiveD is **fault-tolerant**. It can handle random failures of both the scheduler itself and the resource it manages.
 
 ## Feature
-1. [Multi-Tenant: Virtual Cluster (VC)](example/feature/README.md#VC-Safety)
-2. [Fine-grained VC Quota Guarantee (VC Safety)](example/feature/README.md#VC-Safety): HW Quantity, [Topology](example/feature/README.md#VC-Safety), [Type](example/feature/README.md#GPU-Type), [Reservation](example/feature/README.md#Reservation), etc.
-3. Flexible Intra-VC Job Scheduling: HW Quantity, [Topology](example/feature/README.md#Topology-Aware-Scheduling), [(Any) Type](example/feature/README.md#GPU-Type), [Reservation](example/feature/README.md#Reservation), Customization, etc
-4. Few Resource Fragmentation and Starvation
-5. [Priority](example/feature/README.md#Guaranteed-Job), [Overuse](example/feature/README.md#Opportunistic-Job) and [Inter](example/feature/README.md#Inter-VC-Preemption)/[Intra-VC Preemption](example/feature/README.md#Intra-VC-Preemption)
+1. [Multi-Tenancy: Virtual Cluster (VC)](example/feature/README.md#VC-Safety)
+2. [Fine-Grained VC Resource Guarantee](example/feature/README.md#VC-Safety): Quantity, [Topology](example/feature/README.md#VC-Safety), [Type](example/feature/README.md#GPU-Type), [Static Reservation](example/feature/README.md#Reservation), etc.
```

Comment:
>[Static Reservation] [](start = 181, length = 20)

Reservation is enough? And align to other places. 
"Static Reservation" maybe harder to understand for new comer, "I can understand Reservation, but what is static?"